pom.xml

```diff
@@ -502,6 +502,8 @@
           <format>OCI</format>
           <creationTime>USE_CURRENT_TIMESTAMP</creationTime>
           <entrypoint>${cryostat.entrypoint}</entrypoint>
+          <mainClass>io.cryostat.Cryostat</mainClass>
+          <extraClasspath>${env.CLASSPATH}</extraClasspath>
```

Comment:
I'm not sure about this line - if we don't add this, will the Jib-generated classpath file still exist and be populated with default values? If so then I think that would be sufficient. Adding arbitrary things to that classpath based on the host/builder's environment doesn't seem necessary right now.

Reply:
Resolving this conversation as Jib generates the same `CLASSPATH` by default